libs/screen-orientation/src/screen.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Inject, Injectable} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {fromEvent, Observable} from 'rxjs';
+import {map, shareReplay, startWith} from 'rxjs/operators';
+
+@Injectable({
+    providedIn: `root`,
+})
+export class ScreenOrientationService extends Observable<ScreenOrientation> {
+    private readonly stream$ = fromEvent(this.win.screen.orientation, `change`).pipe(
```

Comment:
Would `fromEvent(undefined, 'change')` throw on older browsers? We should probably throw RxJS error if it's unsupported or make a `SUPPORTED` token to people can check it.

Reply:
fixed